bucket/istioctl.json

```diff
@@ -0,0 +1,26 @@
+{
+    "version": "1.16.1",
+    "description": "Configuration command line utility for service operators to debug and diagnose their Istio mesh.",
+    "homepage": "https://istio.io/latest/docs/reference/commands/istioctl/",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/istio/istio/releases/download/1.16.1/istioctl-1.16.1-win.zip",
```

Comment:
What's the difference between this and https://github.com/istio/istio/releases/download/1.16.1/istio-1.16.1-win.zip?

Reply:
A bunch of manifest files, this is the correct download for `istioctl`.